APIs/AuthorizationAPI.raml

```diff
@@ -0,0 +1,139 @@
+#%RAML 1.0
+
+# AMWA NMOS BCP-003 Security: NMOS OAuth API with JSON Web Tokens
+# (c) AMWA 2019
+
+title: NMOS Oauth
+baseUri: http://api.example.com/x-nmos/oauth/{version}
```

Comment:
I'm guessing that this path will need changing from oauth to auth to match other changes. Similarly some cases of OAuth in descriptions probably need to change to Authorization.